External/TestDataSet/dataPar.json.json

```diff
@@ -1,6 +1,7 @@
 {
 	"x": [{
 			"name": "ExampleDataSet",
+			"D": {"ROOT": ".\\derivatives\\ExploreASL"},
 			"subject_regexp": "^Sub-\\d{3}$",
```

Comment:
We should rename `Sub` here to `sub`, throughout, per BIDS

Reply:
Done